assignments/lambda-classes.js

```diff
@@ -1 +1,184 @@
-// CODE here for your Lambda Classes
+// CLASSES
+
+class Person {
+    constructor(data) {
+        this.name = data.name;
+        this.age = data.age;
+        this.location = data.location;
+        this.gender = data.gender;
+    };
+    speak() {
+        console.log(`Hello my name is ${this.name}, I am from ${this.location}`);
+    };
+};
+
+class Instructor extends Person {
+    constructor(data) {
+        super(data);
+        this.specialty = data.specialty;
+        this.favLanguage = data.favLanguage;
+        this.catchPhrase = data.catchPhrase;
+    };
+    demo(subject) {
+        console.log(`Today we are learning about ${subject}`);
+    };
+    gradeStudent(student, subject) {
```

Comment:
Great job here. How could you ensure a grade never fell below zero and went above 100?